include/swift/ClangImporter/ClangImporter.h

```diff
@@ -60,11 +60,28 @@ class TypeDecl;
 class VisibleDeclConsumer;
 enum class SelectorSplitKind;
 
+/// This interface is implemented by LLDB to serve as a fallback when Clang
+/// modules can't be imported from source in the debugger.
+///
+/// During compile time, ClangImporter-imported Clang modules are compiled with
+/// -gmodules, which emits a DWARF rendition of all types defined in the module
+/// into the .pcm file. On Darwin, these types can be collected by
+/// dsymutil. This delegate allows DWARFImporter to ask LLDB to look up a Clang
+/// type by name, synthesize a Clang AST from it. DWARFImporter then hands this
+/// Clang AST to ClangImporter to import the type into Swift.
+class DWARFImporterDelegate {
+public:
+  virtual ~DWARFImporterDelegate() {}
```

Comment:
Nitpick: `= delete` is ever so slightly more optimizable.

Also, please add an "anchor" virtual function to [avoid emitting the vtable in multiple files](https://www.llvm.org/docs/CodingStandards.html#provide-a-virtual-method-anchor-for-classes-in-headers).

Reply:
Gah. As you probably remember I actually had fixed this in between and must have accidentally reverted the fix while refactoring. Thanks for noticing!

Reply:
https://github.com/apple/swift/pull/26675